lib/asset/blogArticle.js

```diff
@@ -13,11 +13,42 @@ const { ensureCheerio, fixLinkTargets } = require('./util');
 
 /** */
 class BlogArticle extends BaseAsset {
-    constructor() {
-        super();
+    constructor(fields) {
+        super(fields);
+
         this._object_type = 'ArticleObject';
         this._content_type = 'text/html';
-        this._tags = [];
+    }
+
+    /**
+     * Set multiple fields at once.
+     * @param {Object} fields - Contains See each setter below for a
```

Comment:
Typo: remove `Contains`

Reply:
Optionally, it might give better documentation to do something like
```
 * @param {string} fields.author - See @link{set_author}
 * @param {string} fields.title - See @link{BaseAsset.set_title}
```
etc.

Reply:
Thanks for noticing.